frontend-implicaction/src/app/forums/components/custom-table-with-search-bar/custom-table-with-search-bar.component.html

```diff
@@ -25,15 +25,15 @@ <h6 class="title">{{ title }}</h6>
       </ul>
     </div>
     <div class="content">
-      <div *ngIf="title && title === 'Forums'">
+      <div *ngIf="tableType.label === 'Forums'">
```

Comment:
Ben du coup c'est sur le code que tu vas checker dsl 😅 .
```suggestion
      <div *ngIf="tableType.code === 'Forums'">
```